corehq/apps/cloudcare/views.py

```diff
@@ -106,11 +111,19 @@ def get(self, request, domain, urlPath):
         if not preview:
             apps = get_cloudcare_apps(domain)
             if request.project.use_cloudcare_releases:
-                # replace the apps with the last starred build of each app, removing the ones that aren't starred
-                apps = filter(
-                    lambda app: app.is_released,
```

Comment:
i don't think we need this anymore because `get_latest_releasd_app_doc` de facto has `is_released` set to `True` unless i'm missing something. this assumption was true on my domain